src/controllers/userController.js

```diff
@@ -0,0 +1,30 @@
+/*
+ * *******************************************************************************
+ *  * Copyright (c) 2018 Edgeworx, Inc.
+ *  *
+ *  * This program and the accompanying materials are made available under the
+ *  * terms of the Eclipse Public License v. 2.0 which is available at
+ *  * http://www.eclipse.org/legal/epl-2.0
+ *  *
+ *  * SPDX-License-Identifier: EPL-2.0
+ *  *******************************************************************************
+ *
+ */
+
+const logger = require('../logger');
+
+const UserService = require('../services/userService');
+
+const userSignupEndPoint = async function (req) {
+  logger.info("Endpoint hit: " + req.originalUrl);
```

Comment:
We can use `morgan` and integrate it with `Express` and `winston` to log requests:
`https://www.digitalocean.com/community/tutorials/how-to-use-winston-to-log-node-js-applications`